src/erlcloud_s3.erl

```diff
@@ -467,9 +468,17 @@ extract_acl(ACL) ->
     [extract_grant(Item) || Item <- ACL].
 
 extract_grant(Node) ->
-    [{grantee, extract_user(xmerl_xpath:string("Grantee", Node))},
+    [{grantee, extract_grantee(hd(xmerl_xpath:string("Grantee", Node)))},
      {permission, decode_permission(erlcloud_xml:get_text("Permission", Node))}].
 
+extract_grantee(Node) ->
```

Comment:
erlcloud_rds uses `lists:map()` when working with xmerl returns...

What i'm trying to ask you here is to eliminate use of `hd()`. You can do it also at line 471 via
```erlang
[Grantee] = xmerl_xpath:string("Grantee", Node),
[{grantee, extract_grantee(Grantee)},
```